sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -221,4 +221,40 @@ class DataFrameAggregateSuite extends QueryTest with SharedSQLContext {
       emptyTableData.agg(sumDistinct('a)),
       Row(null))
   }
+
+  test("moments") {
+    checkAnswer(
+      testData2.agg(skewness('a)),
+      Row(0.0))
+
+    checkAnswer(
```

Comment:
This test needs a tolerance added in order for it to pass. Appreciate any tips on how to do this.